examples/fedora/rpminspect.fmf

```diff
@@ -0,0 +1,44 @@
+/local:
+    summary:
+        Generic rpminspect analysis test on localhost.
+        The rpminspect tool takes care of downloading the artifact.
+    provision:
+        how: local
+    prepare:
+        how: shell
+        script: sudo dnf install -y rpminspect
+    execute:
+        how: shell
+        # Note: script MUST be passed to 'tmt run' on command line
+        # e.g. tmt run -a execute --script "rpminspect BUILD_NVR"
+
+/container:
+    summary:
+        Generic rpminspect comparison test in a prepared container with profile 'gating'.
+        The rpminspect tool takes care of downloading the artifacts.
+    provision:
+        how: container
+        image: rpminspect-container-name
+    execute:
+        how: shell
+        # Note: script MUST be passed to 'tmt run' on command line
+        # e.g. tmt run -a execute --script "rpminspect BEFORE_NVR AFTER_NVR"
+
+/container-with-download:
+    summary:
+        Generic rpminspect comparison test in a prepared container with profile 'gating'.
+        Uses download prepare step to download the artifacts.
+    provision:
+        how: container
+        image: rpminspect-container-name
+    prepare:
```

Comment:
Thinking about the actual usage: Do you expect that specifying a concrete artifact as part of the plan would be a common use case? I would expect this is something that would be almost always provided from the command line. On the other hand it makes sense to keep both L2 keys and command line options consistent.